flang/lib/Optimizer/CodeGen/BoxedProcedure.cpp

```diff
@@ -274,10 +274,10 @@ class BoxedProcedurePass
             auto loc = embox.getLoc();
             mlir::Type i8Ty = builder.getI8Type();
             mlir::Type i8Ptr = builder.getRefType(i8Ty);
-            // For AArch64, PPC32 and PPC64, the thunk is populated by a call to
+            // For PPC32 and PPC64, the thunk is populated by a call to
             // __trampoline_setup, which is defined in
             // compiler-rt/lib/builtins/trampoline_setup.c and requires the
-            // thunk size greater than 32 bytes.  For RISCV and x86_64, the
+            // thunk size greater than 32 bytes.  For Aarch64, RISCV and x86_64, the
             // thunk setup doesn't go through __trampoline_setup and fits in 32
```

Comment:
Is `32 bytes` comment correct for AArch64?  It looks like below it says 36 bytes.

Reply:
The trampoline is now smaller than it was before (28 bytes used, with 4 padding bytes), so while 36 is still acceptable, 32 is correct